### PR TITLE
Implement profile page

### PR DESCRIPTION
**Pull request by joedixon**
Screenshot 2021-08-27 at 16 24 31

Screenshot 2021-08-27 at 16 24 14

Screenshot 2021-08-27 at 16 27 28

Screenshot 2021-08-27 at 16 23 56

### CODE REVIEW

**Review by driesvints — CHANGES_REQUESTED** [review]
Maybe we can do something fun for these places where there's no content?

Screen Shot 2021-08-28 at 11 51 21

Something like...

Screen Shot 2021-08-28 at 11 52 59

I think there are some free artwork platforms that we could consult.

I also realized that the list of replies on the user profile don't make much sense atm. You can't see to which thread they belong. I think we need to slightly redesign these. They also shouldn't be interactive (no clickable likes, solution chosing, no dropdown menu). Just clickable so you can link to them in a thread.

Screen Shot 2021-08-28 at 12 01 21

File: resources/svg/hammer.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg class="w-64 h-64" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 613 613" enable-background="new 0 0 613 613" xml:space="preserve" fill="currentColor">
```
[the remark this answers is not in the document]
Comment (joedixon) [reply resources/svg/hammer.svg:1]: Oops, this is how I pasted it from the blade ui kit icon search and forgot to remove.

**Comment by joedixon** [comment]
> Maybe we can do something fun for these places where there's no content?

I think this is a nice idea, but are you happy if we add it as an issue for now while we find some artwork we like?

**Comment by joedixon** [comment]
New reply component for the profile page\

Screenshot 2021-09-01 at 20 07 36

**Comment by driesvints** [comment]
@joedixon for the empty states. Can you maybe use the "simple" component from https://tailwindui.com/components/application-ui/feedback/empty-states ?

**Comment by joedixon** [comment]
Screenshot 2021-09-02 at 19 56 32

Screenshot 2021-09-02 at 19 56 24

**Pull request merged**